tests/BrowscapTest/Coverage/ProcessorTest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+/**
+ * Copyright (c) 1998-2017 Browser Capabilities Project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Refer to the LICENSE file distributed with this package.
+ *
+ * @category   BrowscapTest
+ * @copyright  1998-2017 Browser Capabilities Project
+ * @license    MIT
+ */
+
+namespace BrowscapTest\Coverage;
+
+use Browscap\Coverage\Processor;
+
+/**
+ * Class ExpanderTest
+ *
+ * @category   BrowscapTest
+ * @author     Jay Klehr <jay.klehr@gmail.com>
+ */
+class ProcessorTest extends \PHPUnit\Framework\TestCase
```

Comment:
Yes please; when I get some time I'd like to fix all sorts of little things like this out :)